File: python/sprint5/K/code.py
# ! change LOCAL to False before submitting !
# set LOCAL to True for local testing

LOCAL = True

if LOCAL:
    class Node:  
        def __init__(self, left=None, right=None, value=0):  
            self.right = right
            self.left = left
            self.value = value

def print_range(node, l, r):
    #  Your code
    #  “ヽ(´▽｀)ノ”
    pass

def test():
    node1 = Node(None, None, 2)
    node2 = Node(None, node1, 1)
    node3 = Node(None, None, 8)
    node4 = Node(None, node3, 8)
    node5 = Node(node4, None, 9)
    node6 = Node(node5, None, 10)
    node7 = Node(node2, node6, 5)
    print_range(node7, 2, 8)
    # expected output: 2 5 8 8

if __name__ == '__main__':
    test()

File: js/sprint5/K/code.js
if (process.env.REMOTE_JUDGE !== 'true') {
    class Node {
        constructor(value, left = null, right = null) {
            this.value = value;
            this.left = left;
            this.right = right;
        }
    }
}

function printRange(root, left, right) {
    // Your code
    // “ヽ(´▽｀)ノ”
}

function test() {
    var node1 = new Node(2, null, null);
    var node2 = new Node(1, null, node1);
    var node3 = new Node(8, null, null);
    var node4 = new Node(8, null, node3);
    var node5 = new Node(9, node4, null);
    var node6 = new Node(10, node5, null);
    var node7 = new Node(5, node2, node6);
    printRange(node7, 2, 8);
    // expected output: 2 5 8 8
}